pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -659,7 +666,8 @@ func (m *kubeGenericRuntimeManager) killContainersWithSyncResult(pod *v1.Pod, ru
 			killContainerResult := kubecontainer.NewSyncResult(kubecontainer.KillContainer, container.Name)
 			if err := m.killContainer(pod, container.ID, container.Name, "", gracePeriodOverride); err != nil {
 				killContainerResult.Fail(kubecontainer.ErrKillContainer, err.Error())
-				klog.Errorf("killContainer %q(id=%q) for pod %q failed: %v", container.Name, container.ID, format.Pod(pod), err)
+				klog.ErrorS(err, "kill container failed", "pod", klog.KObj(pod), "podUID", pod.UID,
```

Comment:
```suggestion
				klog.ErrorS(err, "Kill container failed", "pod", klog.KObj(pod), "podUID", pod.UID,
```

Reply:
Sorry for reply too later, Recently we were in the Chinese New Year holiday. I have changed the code.